src/components/policy/policy_regular/src/sql_pt_queries.cc

```diff
@@ -876,14 +879,17 @@ const std::string kInsertApplicationFull =
     " `default_hmi`, `priority_value`, `is_revoked`, `is_default`, "
     " `is_predata`, `memory_kb`, `heart_beat_timeout_ms`, "
     " `certificate`, `hybrid_app_preference_value`, `endpoint`, `enabled`, "
-    " `auth_token`, `cloud_transport_type`, `icon_url`) "
-    "  VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)";
+    " `auth_token`, `cloud_transport_type`, `icon_url`, "
+    "`allow_unknown_rpc_passthrough`) "
+    "  VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)";
 
 const std::string kSelectApplicationFull =
     "SELECT `keep_context`, `steal_focus`, `default_hmi`, `priority_value`, "
     "  `is_revoked`, `is_default`, `is_predata`, `memory_kb`,"
     "  `heart_beat_timeout_ms`, `certificate`, `hybrid_app_preference_value`, "
-    "  `endpoint`, `enabled`, `auth_token`, `cloud_transport_type`, `icon_url` "
+    "  `endpoint`, `enabled`, `auth_token`, `cloud_transport_type`, "
+    "`icon_url`, "
+    "  `allow_unknown_rpc_passthrough` "
```

Comment:
@jacobkeeler  will we update preloaded policy?

Reply:
I dont think there is a need for this. If the field is missing it will default to false.